tests/testthat/test_redist_plans.R

```diff
@@ -0,0 +1,64 @@
+test_that("constructor works", {
+    fl = redist_map(fl25, ndists=3, pop_tol=0.1)
+    x = redist_plans(plans_10, fl, "enumpart")
+    expect_s3_class(x, "redist_plans")
+    expect_s3_class(x, "tbl")
+    expect_true(is.matrix(as.matrix(x)))
+    expect_equal(attr(x, "prec_pop"), fl25$pop)
+    expect_null(weights(x))
+    expect_output(print(x), "drawn using Enumpart", fixed=T)
```

Comment:
Should `Enumpart` be uppercase? Line 3 has lowercase?

Reply:
yeah it gets capitalized in the output